style.css

```diff
@@ -71,6 +71,14 @@ body {
   border-radius: 50%;
 }
 
+.desktop {
+  display: none !important;
```

Comment:
- Hey, let's find a better way of applying this rule in achieving what we want as overusing the `!important` rule is regarded as a bad practice in the microverse  [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), and as such, you shouldn't use it in your CSS rule except using a CSS framework, let' find a better way of applying the rule.